packages/configs/default/index.json

```diff
@@ -40,6 +40,7 @@
     "style:*.vue": ["@parcel/transformer-vue"],
     "custom:*.vue": ["@parcel/transformer-vue"],
     "*.{png,jpg,jpeg,webp}": ["@parcel/transformer-image"],
+    "*.svg": ["@parcel/transformer-svg"],
```

Comment:
This means that `import img from "./logo.svg";` doesn't cause a build error anymore (currently it needs to be `from "url:./logo.svg"`). But at the moment, there's nothing we can do about that